ui_advanced_config.py

```diff
@@ -8,6 +8,9 @@
 import json
 import comp
 import obj
+import zmap
```

Comment:
I managed to mess it up enough that no objects or maps appear to exist even though I have not attempted  deleting either. (For context, I messed around first, then restarted MAIA to see how it effected data. As far as I know, there were values being displayed before restarting MAIA)
<img width="1218" alt="Screen Shot 2024-03-02 at 1 27 26 AM" src="https://github.com/Leading-AI-Development-Solutions/MAIA/assets/100381732/18fc60b8-9edb-4c31-8ae6-4cbccdc8318e">

Reply:
This is the error associated with it, if it's useful.
<img width="607" alt="Screen Shot 2024-03-02 at 1 31 07 AM" src="https://github.com/Leading-AI-Development-Solutions/MAIA/assets/100381732/0157a854-f1ce-4b7b-a5fe-2bbae24cfc15">

Reply:
Steps for exactly what you did to cause this would be helpful, once again I'm guessing that adding a bunch of junk data and/or likely being able to modify values in ways they shouldn't be modified is causing this issue.